From 1f29355926b56f18bdccaf3de2ce2381e01056c9 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Fri, 15 Apr 2016 09:06:22 -0400
Subject: [PATCH 09/62] anaconda: disable ntp

The systemd has timesyncd to replace anaconda's NTP(Network Time Protocal).

Upstream-Status: Inappropriate [wrlinux specific]
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 anaconda | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/anaconda b/anaconda
index 488656a..8ae562d 100755
--- a/anaconda
+++ b/anaconda
@@ -884,7 +884,6 @@ if __name__ == "__main__":
 
     from pyanaconda import vnc
     from pyanaconda import kickstart
-    from pyanaconda import ntp
     from pyanaconda import keyboard
     from pyanaconda.iutil import ProxyString, ProxyStringError
 
@@ -1333,14 +1332,6 @@ if __name__ == "__main__":
         geoloc.init_geolocation(provider_id=provider_id)
         geoloc.refresh()
 
-    # setup ntp servers and start NTP daemon if not requested otherwise
-    if can_touch_runtime_system("start chronyd"):
-        if anaconda.ksdata.timezone.ntpservers:
-            ntp.save_servers_to_config(anaconda.ksdata.timezone.ntpservers)
-
-        if not anaconda.ksdata.timezone.nontp:
-            iutil.start_service("chronyd")
-
     # FIXME:  This will need to be made cleaner once this file starts to take
     # shape with the new UI code.
     anaconda._intf.setup(ksdata)
-- 
2.7.4

